src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -8181,52 +8181,174 @@ public class Test : ITest
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
-        public async Task GenericInterfaceNotNull1()
+        public async Task TestAllowNullAttributeOnNonNullableReferenceType()
         {
-            await TestInRegularAndScriptAsync(
-@$"#nullable enable 
+            await TestInRegularAndScript1Async(
+$@"#nullable enable
 
 using System.Diagnostics.CodeAnalysis;
 
 {NullableAttributesCode}
 
-interface IFoo<T>
+interface IInterface<T>
 {{
-    [return: NotNull]
```

Comment:
Did we lose the tests on the [NotNull] case?

Reply:
I can't remember why I didn't add tests for [NotNull] on reference types. I'll add those.